app/views/orders/_form.html.erb

```diff
@@ -18,7 +18,7 @@
 
   <div>
     <%= form.label :product_count, style: "display: block" %>
-    <%= form.number_field :product_count, in: 1..1000 %>
+    <%= form.text_field :product_count, in: 1..1000 %>
```

Comment:
I'm a bit surprised this got changed back?

Reply:
oh, sorry, no, it happened because of merge conflict I faced, and I was trying to solve that conflict and most probably returned back the changes. I will push updated version.